source/appModules/devenv.py

```diff
@@ -473,3 +486,36 @@ class IVsTextManager(comtypes.IUnknown):
     COMMETHOD([], HRESULT, 'SetTopLine',
               ( ['in'], c_int, 'iBaseLine' )),
 ]
+
+class UIATextInfoForVisualStudio( UIATextInfo):
+
+	def __init__(self,obj,position,_rangeObj=None):
+		super(UIATextInfoForVisualStudio,self).__init__(obj,position,_rangeObj)
+		self.digitsAtStartOfLineMatch = re.compile(r"^[\d]+\s")
+
+	def _getFormatFieldAtRange(self,range,formatConfig):
+		formatField = super( UIATextInfoForVisualStudio , self)._getFormatFieldAtRange(range,formatConfig)
+		currentElement = self.NVDAObjectAtStart

```

Comment:
Is there any reason you're fetching this before you know whether there's a line number? If there's no line number, this would be a pointless call. Normally, this wouldn't matter, but when it comes to text navigation and cross-process calls, any unnecessary calls are a potential performance problem.